parm/stage/ice_control_RT.yaml.j2

```
@@ -1,7 +1,7 @@
 ice_control_RT:
     mkdir:
         - "{{ COMOUT_ICE_ANALYSIS_MEM }}"
-    copy:
+    link:
```

```suggestion
    link_req:
```
Use `link_req` to ensure that it checks the file is in place.